package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "mattermost-plugin-starter-template",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {}
+}
```

Comment:
This file can be removed

Reply:
We want to remove the whole package-lock.json?

Reply:
I think you may have accidentally run `npm i` in the root of the repository, which created this file. `webapp/package-lock.json` should be kept, but this file can be removed.

Reply:
AHAH! I did keep doing that. I'll go ahead and delete it.